layer1/executor/tasks/tests/snapshot_test.go

```diff
@@ -0,0 +1,450 @@
+//go:build integration
+
+package tests
+
+import (
+	"context"
+	"fmt"
+	"github.com/alicenet/alicenet/consensus/objs"
+	"github.com/alicenet/alicenet/crypto"
+	"github.com/alicenet/alicenet/crypto/bn256"
+	dkgState "github.com/alicenet/alicenet/layer1/executor/tasks/dkg/state"
+	"github.com/alicenet/alicenet/layer1/executor/tasks/snapshots"
+	"github.com/alicenet/alicenet/layer1/executor/tasks/snapshots/state"
+	"github.com/alicenet/alicenet/layer1/tests"
+	"github.com/alicenet/alicenet/layer1/transaction"
+	"github.com/alicenet/alicenet/logging"
+	"github.com/alicenet/alicenet/test/mocks"
+	"github.com/alicenet/alicenet/utils"
+	"github.com/ethereum/go-ethereum/common"
+	ethCrypto "github.com/ethereum/go-ethereum/crypto"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"math/big"
+	"math/rand"
+	"testing"
+	"time"
+)
+
+func Test_SnapshotTask(t *testing.T) {
+	n := 5
+	fixture, suite := CompleteEthDkgCeremony(t, n)
+	eth := fixture.Client
+	ctx := context.Background()
+
+	bnSigners := make([]*crypto.BNGroupSigner, 0)
+	for idx := 0; idx < n; idx++ {
+		dkgState, err := dkgState.GetDkgState(suite.DKGStatesDbs[idx])
+		require.Nil(t, err)
+		signer := &crypto.BNGroupSigner{}
+		err = signer.SetPrivk(dkgState.GroupPrivateKey.Bytes())
+		require.Nil(t, err)
+		bnSigners = append(bnSigners, signer)
+		groupKey, err := bn256.MarshalBigIntSlice(dkgState.MasterPublicKey[:])
+		if err != nil {
+			t.Fatal(err)
+		}
+		err = signer.SetGroupPubk(groupKey)
+		require.Nil(t, err)
+	}
+
+	// Valid at 1024
+	dkgState, err := dkgState.GetDkgState(suite.DKGStatesDbs[0])
+	require.Nil(t, err)
+	grpSig1024, bClaimsBin1024, err := GenerateSnapshotData(1337, 1024, bnSigners, n, dkgState.MasterPublicKey[:], false)
```

Comment:
Should consider setting constants with descriptive names for magic numbers 1337 and 1024.

Reply:
done